sdk/communication/tests.yml

```diff
@@ -28,15 +28,33 @@ extends:
       SKIP_PHONENUMBER_LIVE_TESTS: TRUE
       TEST_PACKAGES_ENABLED: ${{ parameters.TestPackagesEnabled }}
     PostSteps:
+        - script: >-
```

Comment:
This should go in eng/pipelines so all live tests have it.

Reply:
Is this something that eng. team is going to take care?

Reply:
@paolamvhz lets just do it for all as suggested and make the change directly in live.tests.yml